pkg/cmd/cli/describe/printer.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 var (
-	buildColumns            = []string{"NAME", "TYPE", "FROM", "STATUS", "STARTED"}
+	buildColumns            = []string{"NAME", "TYPE", "FROM", "STATUS", "STARTED", "REASON"}
```

Comment:
In "normal" cases this column will be empty, and will just be adding space in the output. Are you sure?